### DISCUSSION TITLE
i install ventura hava a problem， hacmini3.2.2

### DISCUSSION THREAD

**Discussion by Genmer** [osy86/HaC-Mini#1047] [Q&A]
Is the "Secure Boot" mentioned in the update announcement a setting in bios or a custom setting in hacmini.pkg, or do you want to open "Secure Boot"?

**Comment by Medizinmann-MD** [comment]
It is a setting in hacmini.pkg.
https://osy.gitbook.io/hac-mini-guide/installation-guide/installation

If you do it for the forst time - you might need extra steps...
https://osy.gitbook.io/hac-mini-guide/post-installation/support#cannot-boot-when-secure-boot-is-enabled

You do not need to activate Secure Boot in BIOS!
https://osy.gitbook.io/hac-mini-guide/installation-guide/bios-settings

**Comment by Genmer** [comment]
thanks，I'm done，The reason is the bios setting. I turned off a bios controller, resulting in no Thunder safety level setting.
